skimage/filters/tests/test_gaussian.py

```diff
@@ -82,3 +84,14 @@ def test_guess_spatial_dimensions():
     testing.assert_equal(_guess_spatial_dimensions(im4), 3)
     with testing.raises(ValueError):
```

Comment:
You should include a test to ensure the error you want is raised.

Reply:
`test_output_error` is added for it.